R/FilterState.R

```diff
@@ -364,13 +364,16 @@ FilterState <- R6::R6Class( # nolint
     format = function(indent = 0) {
       checkmate::assert_number(indent, finite = TRUE, lower = 0)
 
-      sprintf(
-        "%sFiltering on: %s\n%1$s  Selected values: %s\n%1$s  Include missing values: %s",
-        format("", width = indent),
-        self$get_varname(deparse = TRUE),
-        paste0(format(self$get_selected(), nsmall = 3), collapse = " "),
-        format(self$get_keep_na())
-      )
+      # List all selected values separated by commas.
+      values <- paste(format(self$get_selected(), nsmall = 3, justify = "none"), collapse = ", ")
+      paste(c(
+        strwrap(sprintf("Filtering on: %s", self$get_varname(deparse = TRUE)), indent = indent),
```

Comment:
I think we could cross the 76 chars limit when the varname is very long

Reply:
we could add the test when the limit is crossed, e.g. width = 10

Reply:
Lines are only broken on whitespace characters, in such a case the first line would be longer.

Reply:
What would be the purpose of this test? An extremely long value name (I think this is what you mean) would cause one line to exceed the desired width. The test would fail but the functionality of "limiting printout width" would be maintained.